### PR TITLE
Update usage docs

### PR DESCRIPTION
**Pull request by drpatelh**
# nf-core/nanoseq pull request

Many thanks for contributing to nf-core/nanoseq!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/nanoseq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/nanoseq)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/nanoseq/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -222,54 +304,54 @@ Barcode kit used to perform the sequencing e.g. `SQK-PBK004`. When `--skip_basec
...
 
 ### `--guppy_gpu_runners`
 
-Number of '--gpu_runners_per_device' used for guppy when using `--guppy_gpu` (default: 6)
+Number of "--gpu_runners_per_device" used for Guppy when using `--guppy_gpu` (default: 6).
 
 ### `--guppy_cpu_threads`
 
-Number of '--cpu_threads_per_caller' used for guppy when using `--guppy_gpu` (default: 1)
+Number of "--cpu_threads_per_caller" used for Guppy when using `--guppy_gpu` (default: 1).
```
Comment (arontommi) [inline docs/usage.md]: isn't it supposed to be 'Number of "--cpu_threads_per_caller" used for Guppy when **NOT** using `--guppy_gpu` (default: 1).'

File: docs/usage.md [hunk 2]
```diff
@@ -222,54 +304,54 @@ Barcode kit used to perform the sequencing e.g. `SQK-PBK004`. When `--skip_basec
...
 
 ### `--gpu_device`
 
-Basecalling device specified to Guppy in GPU mode using `--device` (default: 'auto')
+Basecalling device specified to Guppy in GPU mode using "--device" (default: 'auto').
```
Comment (arontommi) [inline docs/usage.md]: why is the formatting on "--device" not `--device` like the rest of the parameters shown above ?

Comment (arontommi) [reply docs/usage.md]: nevermind, realized what you were doing, smart!  (not a part of parameters for the pipeline but for gpu)

File: docs/usage.md (hunk 1 above)
Comment (drpatelh) [reply docs/usage.md]: No. Its actually a setting that is used by Guppy in GPU mode!

**Comment by drpatelh** [comment]
Thanks @arontommi 👍 😎 Ive also updated the output docs so hopefully things are slightly clearer regarding the usage and output of the pipeline now...

**Pull request merged**